### PR TITLE
 migrate: components_polymer3/tf_line_chart_data_loader

### PR DESCRIPTION
**Pull request by stephanwlee**

### CODE REVIEW

File: tensorboard/components_polymer3/tf_line_chart_data_loader/tf-line-chart-data-loader.html [hunk 1]
```diff
@@ -1,274 +0,0 @@
...
-        setSeriesData(name, data) {
-          this.$.chart.setSeriesData(name, data);
-        },
-
-        setSeriesMetadata(name, metadata) {
-          this.$.chart.setSeriesMetadata(name, metadata);
-        },
-
-        commitChanges() {
-          this.$.chart.commitChanges();
```
Comment (nfelt) [inline tensorboard/components_polymer3/tf_line_chart_data_loader/tf-line-chart-data-loader.html:223]: Do we not need this any more?

File: tensorboard/components_polymer3/tf_line_chart_data_loader/tf-line-chart-data-loader.ts [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+    x._maybeRenderedInBadState = false;
+  }
+  window.cancelAnimationFrame(redrawRaf);
+  window.requestAnimationFrame(internalRedraw);
+}, 100);
+
+// A component that fetches data from the TensorBoard server and renders it into
+// a vz-line-chart.
+@customElement('tf-line-chart-data-loader')
+export class TfLineChartDataLoader<ScalarMetadata> extends DataLoader<
```
Comment (nfelt) [inline tensorboard/components_polymer3/tf_line_chart_data_loader/tf-line-chart-data-loader.ts:55]: Question - where is the `ScalarMetadata` type coming from?  I'm getting an import error when I'm trying to parameterize a TfLineChartDataLoader<> for the scalar dashboard (with your PR patched locally).

Comment (nfelt) [reply tensorboard/components_polymer3/tf_line_chart_data_loader/tf-line-chart-data-loader.ts:55]: Or I see, that's actually a type parameter?  So I guess I need to define the appropriate type on my end.

Comment (stephanwlee) [reply tensorboard/components_polymer3/tf_line_chart_data_loader/tf-line-chart-data-loader.ts:55]: Yup, it is a generic.

**Pull request merged**